src/tests/ftest/control/dmg_telemetry_io_latency.py

```diff
@@ -295,31 +324,17 @@ def test_io_latency_telmetry_metrics(self):
 
         for transfer_size in transfer_sizes:
             ior_latency[transfer_size] = {}
-            self.add_pool(connect=False)
-            oclass = self.ior_cmd.dfs_oclass.value
-            self.add_containers(self.pool, oclass)
             for operation in ["update", "fetch"]:
-                flags = self.params.get("F", "/run/ior/ior{}flags/".format(
-                    operation))
-                self.log.info(
-                    "<<< Start ior %s transfer_size=%s", operation, transfer_size)
-                self.ior_cmd.transfer_size.update(transfer_size)
-                self.ior_cmd.flags.update(flags)
-                self.ior_cmd.set_daos_params(
-                    self.server_group, self.pool, self.container[-1].uuid)
                 # Run ior command
-                ior_results = self.run_ior_with_pool(
-                        timeout=200, create_pool=False, create_cont=False)
+                ior_results = self.ior_with_transfer_size(transfer_size, operation)
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'operation' in method call